.env

```diff
@@ -6,6 +6,7 @@ REACT_APP_GRAPHQL_BASE_URL=https://tapahtumat-proxy.test.kuva.hel.ninja/proxy/gr
 
 REACT_APP_CMS_URL=https://cms.test.kuva.hel.ninja/api
 REACT_APP_LINKED_EVENTS_URL=https://api.hel.fi/linkedevents/v1
+REACT_APP_LINKED_COURSES_URL=https://api.hel.fi/linkedcourses/v1
```

Comment:
In think this variable should also be added to Dockerfile

Reply:
👍

Reply:
added